scripts/compare_healthdata_workaround.R

```diff
@@ -0,0 +1,53 @@
+# This file is meant to AB test the workaround we used in run.R to get the most
+# recent data from healthdata.gov. This requires manually downloading the right
+# file and transforming it in the way the HHS pipeline would. All that is done
+# in `get_health_data`.
+
+source(here::here("R", "load_all.R"))
+
+forecast_generation_date <- as.Date("2023-03-01")
+
+workaround_data <- get_health_data(forecast_generation_date) %>%
+  process_healthdata()
+
+epidatr_data <- bind_rows(
+  epidatr::pub_covidcast(
+    "hhs",
+    "confirmed_admissions_influenza_1d_prop_7dav",
+    "state",
+    "day",
+    "*",
+    epirange(forecast_generation_date - 2000L, forecast_generation_date),
+    as_of = forecast_generation_date
+  ),
+  epidatr::pub_covidcast(
+    "hhs",
+    "confirmed_admissions_influenza_1d_prop_7dav",
+    "nation",
+    "day",
+    "*",
+    epirange(forecast_generation_date - 2000L, forecast_generation_date),
+    as_of = forecast_generation_date
+  )
+)
+
+# Are there differences in the overlapping (time_value, geo_keys)? No.
+full_join(
+  workaround_data,
+  epidatr_data %>% select(geo_value, time_value, value),
+  by = c("geo_value", "time_value")
```

Comment:
maybe this is just future-proofing now (for when we hypothetically bring back other covariates), but I'd imagine joins should include data_source and signal (this might need added to workaround data)

Reply:
(Looks like we at least set the signal in the workaround data, though both the workaround and nonworkaround branches probably need changes to accommodate auxiliary covariates.  But might be good to fix joins here since we're more likely to overlook it when trying to add covariates.)

Reply:
Let's update these scripts whenever that happens. I'll add a comment pointing this detail out above.